Test.MegaCorpClash.Models/ChatCommandHandlers/TestHireCommandHandler.cs

```diff
@@ -24,57 +24,20 @@ public TestHireCommandHandler()
         });
     }
 
-    [Fact]
-    public void Test_NoCompany()
-    {
-        Dictionary<string, Company> companies = new();
-
-        var commandHandler =
-            new HireCommandHandler(_gameSettings, companies);
-
-        var gameCommand = GetGameCommand("!hire");
-
-        var chatMessageEvent =
-            Assert.Raises<ChatMessageEventArgs>(
-                h => commandHandler.OnChatMessagePublished += h,
-                h => commandHandler.OnChatMessagePublished -= h,
-                () => commandHandler.Execute(gameCommand));
-
-        Assert.NotNull(chatMessageEvent);
-        Assert.Equal(DEFAULT_CHATTER_DISPLAY_NAME, 
-            chatMessageEvent.Arguments.ChatterDisplayName);
-        Assert.Equal(Literals.YouDoNotHaveACompany, 
-            chatMessageEvent.Arguments.Message);
-    }
-
-    [Fact]
-    public void Test_NoParameters()
+    [Theory]
+    [InlineData("NoParameters", "")]
```

Comment:
The "name" parameter is not being used anywhere that I see. Is it needed?

Reply:
I have a comment about this with screen shot just above ... its not needed but adds clarity to what you are testing with in the test runner. I am fine to remove it .. BUT I find a lot of time with tests the hard part is not making them green its answering WHY they are there in the first place a name helps that.

i.e xyc fails buy xyz passes what where we testing with a name that is communicated better imo.

Reply:
if this answers your question feel free to mark this as resolved if not hola at me il remove the name from params